packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -478,4 +487,29 @@ export class ConversationService {
       return this.establishMLS1to1Conversation(groupId, selfUser, otherUserId);
     }
   };
+
+  private async handleMLSMessageAddEvent(event: ConversationMLSMessageAddEvent) {
+    return handleMLSMessageAdd({event, mlsService: this.mlsService});
```

Comment:
In the future, we should remove this injection of the entire `mlsService` to the message handler, there is something a little wrong there. (not for this PR though). 

I feel we have an architectural problem, the handler should not access the entire MLS Service, we should try to find a way to limit what we inject in those. 

It feels weird that handling mls event doesn't go **first** to the `mlsService` that then redirects to the correct handler, that should be the ideal data flow